components/xata/xata.app.mjs

```diff
@@ -0,0 +1,148 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "xata",
+  propDefinitions: {
+    recordId: {
+      type: "string",
+      label: "Record ID",
+      description: "ID of the record to create or update",
+    },
```

Comment:
Could this endpoint be used to get existing record ids?
https://xata.io/docs/api-reference/db/db_branch_name/tables/table_name/query#query-table